webapp/client/src/pages/FahrkostenpauschalePersonBPage.js

```diff
@@ -0,0 +1,39 @@
+import PropTypes from "prop-types";
+import React from "react";
+import { fieldPropType, stepHeaderPropType } from "../lib/propTypes";
+import FahrkostenpauschalePage from "./FahrkostenpauschalePage";
+
+export default function FahrkostenpauschalePersonBPage({
+  stepHeader,
+  form,
+  fields,
+  prevUrl,
+  fahrkostenpauschaleAmount,
+}) {
+  return (
+    <FahrkostenpauschalePage
+      {...{ stepHeader, form, prevUrl, fahrkostenpauschaleAmount }}
+      fields={{
+        requestsFahrkostenpauschale: {
+          ...fields.requestsFahrkostenpauschale,
+          name: "person_b_requests_fahrkostenpauschale",
+        },
+      }}
+    />
+  );
+}
+
+FahrkostenpauschalePersonBPage.propTypes = {
+  stepHeader: stepHeaderPropType.isRequired,
+  form: PropTypes.exact({
+    action: PropTypes.string,
+    csrfToken: PropTypes.string,
+    showOverviewButton: PropTypes.bool,
+    nextButtonLabel: PropTypes.string,
+  }).isRequired,
+  fields: PropTypes.exact({
+    requestsFahrkostenpauschale: fieldPropType,
```

Comment:
I think this should be named `personBRequestsPauschbetrag` because of the InputForm mapping.